src/app/(site analysis)/sites/[siteId]/reports/[reportId]/edit/edit-sections.tsx

```diff
@@ -113,21 +114,22 @@ function UpdateSiteReportSection({
   });
 
   return (
-    <Card>
+    
+    <Card className="p-4">
       <CardContent className="p-6">
         <Form {...form}>
           <form
             onSubmit={form.handleSubmit((data) => {
               console.log(data);
               Actions.updateSiteReportSection(section.id, data);
             })}
-            className="grid grid-cols-3 gap-4"
+            className="flex-col"
           >
             <FormField
               control={form.control}
               name="title"
               render={({ field }) => (
-                <FormItem>
```

Comment:
it's better to use gap-4 in the parent element rather than adding padding to each child element